server/hotrod/src/main/java/org/infinispan/server/hotrod/tx/operation/RollbackTransactionOperation.java

```diff
@@ -143,8 +143,7 @@ CacheRpcCommand buildForwardCommand(ByteString cacheName, long timeout) {
 
    @Override
    CompletableFuture<Void> asyncCompleteLocalTransaction(AdvancedCache<?, ?> cache, long timeout) {
```

Comment:
```suggestion
   CompletionStage<Void> asyncCompleteLocalTransaction(AdvancedCache<?, ?> cache, long timeout) {
```

Reply:
👍

Reply:
Was there an issue with `CompletionStage` here?

Reply:
Nope, I updated the parent one and missed this one.